src/sentry/utils/auth.py

```diff
@@ -298,3 +298,10 @@ def authenticate(self, username=None, password=None):
                 except ValueError:
                     continue
         return None
+
+    # TODO(joshuarli): When we're fully on Django 1.10, we should switch to
+    # subclassing AllowAllUsersModelBackend (this isn't available in 1.9 and
+    # simply overriding user_can_authenticate here is a lot less verbose than
+    # conditionally importing).
+    def user_can_authenticate(self, user):
```

Comment:
Just noting this doesn't exist in 1.9, so does nothing until we upgrade to 1.10 (just in case others are confused like I was).